cmd/skopeo/utils.go

```diff
@@ -108,6 +117,19 @@ func imageFlags(global *globalOptions, shared *sharedImageOptions, flagPrefix, c
 	return fs, opts
 }
 
+type retryOptions struct {
+	maxRetry   int           // The number of times to retry
+	retryDelay time.Duration // Time to wait when only retry once
+}
+
+func retryFlags() (pflag.FlagSet, *retryOptions) {
+	opts := retryOptions{}
+	fs := pflag.FlagSet{}
+	fs.IntVar(&opts.maxRetry, "retry-times", 0, "the number of times to retry, retry wait time will be exponential backoff delay based on the time of failed attempts")
+	fs.DurationVar(&opts.retryDelay, "initial-retry-delay", time.Second, "the initial delay peroid and is used to calculate the rest delays between reties")
```

Comment:
Maybe
```suggestion
	fs.DurationVar(&opts.retryDelay, "initial-retry-delay", time.Second, "the initial delay period, subsequent retries use longer delays")
```
but I’ll definitely defer to native speakers and people with more experience writing documentation.